README.md

```diff
@@ -85,6 +85,12 @@ While multiple TYNDP features are already introduced to the Open-TYNDP model, th
 |                                                        |                  Hybrid heat pumps                  |                                                                                                                                            |   ⌛    |
 |                                                        |     Industrial electricity and hydrogen demands     |                                                                                                                                            |   ⌛    |
 |                                                        |                   Hydrogen zones                    |                                    [#17](https://github.com/open-energy-transition/open-tyndp/pull/17)                                     |   ✅    |
+|             **CBA Benefit Indicators**                 |              B1: Socioeconomic Welfare (SEW)                  |                                                                                                                                  |   ⌛    |
+|                                                        |            B2: CO₂ Variation with societal costs              |                                                                                                                                  |   ⌛    |
+|                                                        |         B3: DRES Integration (curtailment reduction)          |                                                                                                                                  |   ⌛    |
+|                                                        |     B4: Non-CO₂ Emissions (NOx, SOx, PM2.5/10, NMVOC, NH₃)    |                                                                                                                                  |   ⌛    |
+|                                                        | Security of Supply: Adequacy, Flexibility & Stability (B6-B8) |                                                                                                                                  |   ⌛    |
```

Comment:
Remove " Security of Supply: Adequacy, Flexibility & Stability", SoS is already mentioned in the table

It's fine to cluster the remaining indicators, which cannot be run by a market model, but in total they are 9  and we should start with B5-B9